Backpack-SwiftUI/Font/Classes/BPKFont.swift

```diff
@@ -20,10 +20,15 @@ import SwiftUI
 
 public struct BPKFont {
     static var fontDefinition: BPKFontDefinition?
+    static var enableDynamicType: Bool = false
     
     public static func setFontDefinition(_ fontDefinition: BPKFontDefinition) {
         BPKFont.fontDefinition = fontDefinition
     }
+    
+    public static func setDynamicType(enabled: Bool) {
+        BPKFont.enableDynamicType = enabled
+    }
```

Comment:
@frugoman Added this, so we can easily test font scaling if we are building our UI's

Reply:
Nice one!